package.json

```diff
@@ -126,7 +126,7 @@
     "babel-register": "^6.18.0",
     "babili-webpack-plugin": "^0.1.1",
     "buttercup-importer": "~0.9.2",
-    "buttercup-web": "~0.32.1",
+    "buttercup-web": "^0.39.0",
```

Comment:
Probably safer to have this patch-fixed: `~0.39.0`..

Reply:
@perry-mitchell done mate